hubmap/frontend/src/commons/apiAdapter.js

```diff
@@ -0,0 +1,7 @@
+let API_URL
+
+process.env.REACT_APP_STAGE === 'dev'
+    ? API_URL = 'http://localhost:8000/'
+    : API_URL = 'https://data.dev.hubmapconsortium.org/'
```

Comment:
The `? :` ternary is an expression, not a statement: Better to just use the value it returns.
```
const API_URL = process.env.REACT_APP_STAGE === 'dev'
    ? 'http://localhost:8000/'
    : 'https://data.dev.hubmapconsortium.org/'
```